regression/memory-analyzer/Makefile

```diff
@@ -0,0 +1,21 @@
+default: clean tests.log
```

Comment:
Could we also have a CMakeLists.txt, please?

Reply:
Added.